npe2/manifest/contributions/_contributions.py

```diff
@@ -27,5 +27,16 @@ class ContributionPoints(BaseModel):
     # configuration: Optional[JsonSchemaObject]
     # keybindings: Optional[List[KeyBindingContribution]]
 
+    @validator("menus")
+    def _check_napari_menus(cls, v):
+        valid_names = [n.key for n in napari_menus]
```

Comment:
minor nit, this should be a set and probably put at module level (since it won't change  within a session:
```python
VALID_NAMES = {n.key for n in napari_menus}
```

Reply:
+1 this will make it easier to access from the napari side as well